### ISSUE TITLE
HTTP: Proxy support EL

### ISSUE THREAD

**Issue by Amerousful** [gatling/gatling#4265]
Motivation: 
When testing our application's CDN or about how our application collects metrics for a different location, it's useful to be able to pass a list of proxy servers and thus distribute the load from the generator. It isn't always possible to set up load instances in the desired zone/country, and proxies are an easy solution. Since now Gatling does not support EL for the proxy method, you have to do 1 instance - 1 proxy server

**Comment by Amerousful** [comment]
Maybe you're right and this case only mine... 

What about public shared proxies - no, we wouldn't like to buy not public proxies, only dedicated for us. 
Performance between injectors and proxies in our cases doesn't matter. We just created load and took metrics on application side (where we saw response time between application and proxy). 
As well I uped proxy-master which rotates N proxy servers (1k in our case), and in this case master-proxy can be a bottelneck. So I decided to remove master-proxy and just pass all proxy servers as a list
